src/open_inwoner/core/views.py

```diff
@@ -49,56 +56,126 @@ def sitemap(request):
+            {"url_name": "ssd:uitkeringen", "link_text": _("Mijn uitkeringen")}
         )
     if case_page_is_published():
         context["cms_pages"].append(
-            {"url_name": "cases:index", "link_text": "Mijn zaken"}
+            {"url_name": "cases:index", "link_text": _("Mijn zaken")}
         )
         context["cms_pages"].append(
-            {"url_name": "cases:contactmoment_list", "link_text": "Mijn vragen"}
+            {"url_name": "cases:contactmoment_list", "link_text": _("Mijn vragen")}
         )
     if collaborate_page_is_published():
         context["cms_pages"].append(
-            {"url_name": "collaborate:plan_list", "link_text": "Mijn samenwerkingen"}
+            {"url_name": "collaborate:plan_list", "link_text": _("Mijn samenwerkingen")}
         )
     if inbox_page_is_published():
         context["cms_pages"].append(
-            {"url_name": "inbox:index", "link_text": "Mijn berichten"}
+            {"url_name": "inbox:index", "link_text": _("Mijn berichten")}
         )
     if products_page_is_published():
         context["cms_pages"].append(
-            {"url_name": "products:questionnaire_list", "link_text": "Zelftest"}
+            {"url_name": "products:questionnaire_list", "link_text": _("Zelftest")}
         )
     if profile_page_is_published():
-        base_url = reverse("profile:contact_list")
-        begeleiders_url = f"{base_url}?{urlencode({'type': 'begeleider'})}"
-
         context["cms_profile_pages"] = [
-            {"url_name": "profile:detail", "link_text": "Mijn profiel"},
-            {
-                "url_name": "profile:categories",
-                "link_text": "Mijn interessegebieden",
-            },
-            {
-                "url": begeleiders_url,
-                "link_text": "Mijn begeleiders",
-            },
-            {"url_name": "profile:contact_list", "link_text": "Mijn contacten"},
-            {"url_name": "profile:appointments", "link_text": "Mijn afspraken"},
-            {"url_name": "profile:action_list", "link_text": "Openstaande acties"},
+            {"url_name": "profile:detail", "link_text": _("Mijn profiel")},
         ]
 
+        # conditionally add cms pages based on apphook configuration
+        cms_pages = context["cms_profile_pages"]
+        try:
+            profile_config = ProfileConfig.objects.get()
+        except ProfileConfig.MultipleObjectsReturned:
+            logger.warning("Multiple instances of ProfileConfig apphook found")
+            profile_config = ProfileConfig.objects.first()
+        if profile_config.selected_categories:
+            base_url = reverse("profile:contact_list")
+            begeleiders_url = f"{base_url}?{urlencode({'type': 'begeleider'})}"
+            cms_pages.append(
+                {
+                    "url": begeleiders_url,
+                    "link_text": _("Mijn begeleiders"),
+                },
+            )
+        if profile_config.mentors:
```

Comment:
Isn't this the same s Begeleiders above?
i now see two links called Mijn contacten.